usr/share/device-quirks/scripts/asus/ally/ally.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+if [ $(whoami) != 'root' ]; then
+  echo "You must be root to run this script."
+  exit 1
+fi
+
+if [ -d /tmp/frzr_root ]; then
+  source ${MOUNT_PATH}/etc/device-quirks/device-quirks.conf
+else
+  source /etc/device-quirks/device-quirks.conf
+fi
+
+# Create an equalizer for pipwier.
```

Comment:
Small typo for pipewire here ...

Reply:
ah. . Embarrassingly I had a typo.:)
I have corrected the mistakes.
Thank you!